common/errors.js

```diff
@@ -4,7 +4,42 @@
     angular.module('chaise.errors', ['chaise.utils', 'chaise.alerts'])
 
     // Factory for each error type
-    .factory('ErrorService', ['$log', 'UriUtils', 'AlertsService', function ErrorService($log, UriUtils, AlertsService) {
+    .factory('ErrorService', ['$log', '$window', '$uibModal', 'UriUtils', 'AlertsService', 'Session', function ErrorService($log, $window, $uibModal, UriUtils, AlertsService, Session) {
+
+        function errorPopup(exception) {
+            $log.info(exception);
+
+            var params = {
+                message: exception.message,
+                errorCode: exception.code
+            }
+
+            var modalInstance = $uibModal.open({
+                templateUrl: '../common/templates/errorDialog.html',
+                controller: 'ErrorDialogController',
+                controllerAs: 'ctrl',
+                backdrop: 'static',
+                keyboard: false,
+                resolve: {
+                    params: function() {
+                        return params;
+                    }
+                }
+            });
+
+            modalInstance.result.then(function () {
+                $window.location = chaiseConfig.dataBrowser ? chaiseConfig.dataBrowser : $window.location.origin;
```

Comment:
If we're truly redirecting (are we? cc @robes?) and not just going to another page, then it should be `$window.location.replace(url)`. [window.location vs window.location.replace()](http://stackoverflow.com/questions/1865837/whats-the-difference-between-window-location-and-window-location-replace)